modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1073,6 +1074,42 @@ return view.extend({
 					    bssid = ss.children[5],
 					    encr;
 
+					/* 802.11v settings start */
+					// Probe 802.11v support (needs full hostapd/wpad) via EAP support (full hostapd has EAP)
+					if(L.hasSystemFeature('hostapd', 'eap'))
+						o = ss.taboption('advanced', form.ListValue, 'time_advertisement', _('Time advertisement'), _('802.11v: Time Advertisement in management frames.'));
+						o.value('0', _('Disabled'));
+						o.value('2', _('Enabled'));
+						o.write = function (section_id, value) {
+							return this.super('write', [section_id, (value == 2) ? value: null]);
+						}
+
+						//Pull current System TZ setting
+						var tz = uci.get('system', '@system[0]', 'timezone');
+						o = ss.taboption('advanced', form.Value, 'time_zone', _('Time zone'), _('802.11v: Local Time Zone Advertisement in management frames.'));
+						o.value(tz);
+						o.rmempty = true;
+						o.write = function (section_id, value) {
+							return this.super('write', [section_id, (value != null) ? value: null]);
+						}
+
+						o = ss.taboption('advanced', form.Flag, 'wnm_sleep_mode', _('WNM Sleep Mode'), _('802.11v: Wireless Network Management (WNM) Sleep Mode (extended sleep mode for stations).'));
+						o.write = function (section_id, value) {
+							return this.super('write', [section_id, (value == true) ? value: null]);
+						}
+
+						o = ss.taboption('advanced', form.Flag, 'wnm_sleep_mode_no_keys', _('WNM Sleep Mode Fixes'), _('802.11v: Wireless Network Management (WNM) Sleep Mode Fixes: Prevents reinstallation attacks.'));
+						/* https://git.openwrt.org/?p=openwrt/openwrt.git;a=commitdiff;h=bf98faaac8ed24cf7d3d93dd4fcd7304d109363b */
+						o.write = function (section_id, value) {
+							return this.super('write', [section_id, (value == true) ? value: null]);
```

Comment:
This `o.write()` override seems to reimplement the default behaviour. Why is it needed?